### ISSUE TITLE
Unintended element updates

### ISSUE THREAD

**Issue by aidenybai** [aidenybai/lucia#34]
**Describe the bug**
Due to https://github.com/aidenybai/lucia/blob/fa0246353c791bb5fced79ab12a1c4bfd8506fe5/src/vdom/patch.ts#L29

If two view keys are provided with one name being inside the other, such as `foo` and `foo1` or `phrase` and `wowaphraseawow`, then it will update the latter if the first is updated. 

**To Reproduce**
1. Run 
```html
<div l-use="{ count: 0, count2: 0 }">
  <button l-text="this.count" l-on:click="++this.count">0</button>
  <button l-text="this.count2" l-on:click="++this.count2">0</button>
</div>
```
2. Click the first button and view the Elements view in Chrome Devtools
3. View the update flashes on elements

**Expected behavior**
The latter view key should not update. Such as `foo` and `foo1`, if `foo` updates, then `foo1` should not.

**Comment by aidenybai** [comment]
Possible fix is to use some sort of regex match? I want to keep it string check based rather than parse it for low overhead

**Comment by aidenybai** [comment]
Fixed in 58aaf17cd5c31a4518d6ebd20d72460199fd3293
